From 7ee479e1e460738be8320891212c68e19b7b1b4d Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 5 Sep 2014 13:08:24 -0400
Subject: [redhat] RHMAINTAINERS: remove fenlason

Message-id: <1409922504-28043-1-git-send-email-prarit@redhat.com>
Patchwork-id: 91446
O-Subject: [RHEL7.1 PATCH INTERNAL] RHMAINTAINERS, remove fenlason
Bugzilla:
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Jay is no longer at Red Hat.  Update RHMAINTAINERS with new maintainers.

Cc: nhorman@redhat.com
Cc: svemuri@redhat.com
Cc: tcamuso@redhat.com
Cc: jarod@redhat.com
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/RHMAINTAINERS b/redhat/RHMAINTAINERS
index b784b80..2cd17cf 100644
--- a/redhat/RHMAINTAINERS
+++ b/redhat/RHMAINTAINERS
@@ -193,12 +193,12 @@ F: drivers/cpufreq/
 F: arch/x86/kernel/cpu/cpufreq/
 
 CXGB3 ETHERNET DRIVER (cxgb3)
-M: Jay Fenlason <fenlason@redhat.com>
+M: Sai Vemuri <svemuri@redhat.com>
 S: Maintained
 F: drivers/net/cxgb3/
 
 CXGB4 ETHERNET DRIVER (cxgb4)
-M: Jay Fenlason <fenlason@redhat.com>
+M: Sai Vemuri <svemuri@redhat.com>
 S: Maintained
 F: drivers/net/cxgb4/
 
@@ -234,7 +234,7 @@ S: Maintained
 F: drivers/net/forcedeth.c
 
 FIREWIRE SUBSYSTEM
-M: Jay Fenlason <fenlason@redhat.com>
+M: Neil Horman <nhorman@redhat.com>
 S: Maintained
 F: drivers/firewire/
 F: include/linux/firewire*.h
@@ -318,7 +318,6 @@ F: drivers/net/ibmveth.*
 
 INFINIBAND SUBSYSTEM
 M: Doug Ledford <dledford@redhat.com>
-M: Jay Fenlason <fenlason@redhat.com>
 S: Maintained
 F: drivers/infiniband/
 
@@ -671,7 +670,7 @@ S: Maintained
 F: drivers/usb/
 
 UVC WEBCAM DRIVER
-M: Jay Fenlason <fenlason@redhat.com>
+M: Tony Camuso <tcamuso@redhat.com>
 S: Maintained
 F: drivers/media/video/uvc/
 
-- 
1.7.1